{"name": "Проверка ошибочного логина", "status": "passed", "start": 1740154080367, "stop": 1740154080585, "uuid": "5770ac44-4756-432d-b07b-90a7e887258c", "historyId": "34c563db843068ac2366f3e8a87d8aaa", "testCaseId": "34c563db843068ac2366f3e8a87d8aaa", "fullName": "tests.test_login.TestLogin#test_incorrect_user_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "GM0V7WMM2R-MBP"}, {"name": "thread", "value": "46293-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}